Dockerfile

```diff
@@ -3,17 +3,18 @@ LABEL maintainer="Charlie Lewis <clewis@iqt.org>"
 
 COPY requirements.txt requirements.txt
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    curl \
-    gcc \
-    git \
-    python3 \
-    python3-dev \
-    tcpdump \
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY . /networkml
```

Comment:
this should be after the apt-get, otherwise this invalidates the cache of everything after it and makes it so you have to run that apt-get every time you build.